[G2M] Plotly/rework legend

vxsl opened the pull request:
- added dep `lumen-labs` to have access to `getTailwindConfigColor()`
  - had to make some adjustments to the workflow files to allow installing the scoped package
- legend is now positioned relatively to the viz, i.e. it will not overlap
- legend styling has generally been changed
- viz size only applies to subplots (defaults to 0.8 otherwise)

Legend styling is still far from perfect. Will keep working on it over time.

geoerika commented on .github/workflows/chromatic.yml:
```diff
@@ -6,18 +6,27 @@ jobs:
+          token: ${{ secrets.CD_GITHUB_TOKEN }}
           projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
           exitZeroOnChanges: true
           autoAcceptChanges: true
```
Can you please add the `  storybook-pr-comment:` job to this as well?

geoerika reviewed (approved):
Looks good! The story titles for the bar charts don't really match with the charts, ie. the stacked bars are not stacked. :)

vxsl commented:
> Looks good! The story titles for the bar charts don't really match with the charts, ie. the stacked bars are not stacked. :)

Oh lol nice catch. Added to the todo

> Can you please add the storybook-pr-comment: job to this as well?

Added to the todo, will do that in time for the next PR
